webrtc.html

```diff
@@ -1807,8 +1807,8 @@
                         </ol>
                       </li>
                       <li>
-                        <p>If <code><var>candidate</var>.ufrag</code> is
-                        non-null, and is not equal to any ufrag present in the
+                        <p>If <code><var>candidate</var>.ufrag</code> is not
+                        missing, and is not equal to any ufrag present in the
```

Comment:
Good point, `candidate` can be either a dictionary or an object here, which is unusual! Even more reason to be explicit IMHO: "If candidate.ufrag is not `undefined` and not `null`", or maybe neither nor?